.clang-format

```diff
@@ -64,7 +64,7 @@ ForEachMacros:
   - foreach
   - Q_FOREACH
   - BOOST_FOREACH
-IncludeBlocks:   Regroup
+IncludeBlocks:   Preserve # TODO Should this be Regroup? How do we feel about it?
```

Comment:
I'd prefer Regroup. Our includes are a mess.

Unless you saw strange outputs from Regroup?

Reply:
No strange output. Just a lot of changes.

Reply:
Yeah, I think this is one case where we WANT a lot of changes :)